### ISSUE TITLE
openseadragon.min.css is needed

### ISSUE THREAD

**Issue by dusko23** [openseadragon/openseadragon#2471]
Hi, I have downloaded distribution files and I am using **openseadragon.min.js** locally.
However, I did not find a file **openseadragon.min.css**  
Could anybody tell if **Openseadragon.min.css** is available as a file?

**Comment by pearcetm** [comment]
There is no css file needed for the OpenSeadragon library - all necessary styling on the created elements is done inline by the javascript file.

**Comment by dusko23** [comment]
Thank you for your answer.
I have removed 
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/2.4.0/openseadragon.min.css" />

and now I don't see icons:

![image](https://github.com/openseadragon/openseadragon/assets/26789352/f8fb02af-35e3-4c40-9a3a-ab3292e22f50)

**Comment by pearcetm** [comment]
You need to specify where the images for the buttons are found using the `prefixUrl` option. See https://openseadragon.github.io/docs/OpenSeadragon.html#.Options. These can be downloaded (see https://openseadragon.github.io/docs/ and https://openseadragon.github.io/#download) or you can give it a URL to a CDN.

**Comment by dusko23** [comment]
Thanks, will do that.

**Comment by dusko23** [comment]
Is it possible to keep images needed by prefixURL locally? Like in a folder on a local computer for pure local testing? 
Later, images will be uploaded to the server.

**Comment by iangilman** [comment]
@dusko23 Yes, you can download the images at the link @pearcetm gave above.
